django-cloudlaunch/cloudlaunch/backend_plugins/base_vm_app.py

```diff
@@ -244,10 +247,65 @@ def resolve_launch_properties(self, provider, cloudlaunch_config):
                 provider, subnet, cloudlaunch_config['firewall'])
         return subnet, placement, vmf
 
-    def launch_app(self, provider, task, name, cloud_config,
-                   app_config, user_data):
-        """Initiate the app launch process."""
+    def _create_ssh_key(self):
```

Comment:
This function is available in cloudbridge.cloud.base.helpers. Can we refactor to use that instead?